samples/snippets/create_kmeans_model_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def test_kmeans_sample():
+# [START bigquery_dataframes_bqml_kmeans]
+    import bigframes.pandas as bpd
+    import bigframes
+    from bigframes import dataframe
```

Comment:
We shouldn't have to import this. It's a private-ish module.

Reply:
Corrected